### PR TITLE
Update docker-compose.md

### PR DESCRIPTION
**Pull request by rajeshkumargp**
Added Local Directory Mapping from Docker-compose itself

### CODE REVIEW

**Comment by tobybatch** [comment]
Thanks for the contribution, I'm not sure what it adds.  Why not just allow the container to inherit the default settings from the docker set up?

Do these setting add anything?

File: docs/docker-compose.md [hunk 1]
```diff
@@ -64,10 +64,19 @@ services:
...
+        device: /.../kimai2/kimaidata/var_data <ABSOLUTE PATH to LOCAL DIRECTORY>
     public:
+      driver: local
+      driver_opts:
+        type: none
+        o: bind
+        device: /.../kimai2/kimaidata/public_data <ABSOLUTE PATH to LOCAL DIRECTORY>
 ```
 
 **You must use a named volume**
```
Comment (Schrolli91) [inline docs/docker-compose.md:82]: If i'am right, you use a bind to the host machine here.
Whats about this problem with bind mounts?

**Comment by tobybatch** [comment]
@Schrolli91 I think this is out of scope for this repo.  We provide a containerised kimai solution with some guidelines about how to get it running.  If @rajeshkumargp wants to write a page for the docs explaining how to use different docker compose settings then I'm really happy to look into that but the docker compose provided is only a guide line.

Any thoughts?

**Comment by Schrolli91** [comment]
@tobybatch yes i can total agree that - better to move something like that in a chapter at the docs to guide people...
The compose file should as simple as possible to guarantee an easy start.

**Comment by tobybatch** [comment]
@rajeshkumargp Thanks for the suggested addition but we've decided that the docker-compose.yml should be kept as simple as possible.  If you want to add this a section in the docs then please add it in the docs folder and re-submit the PR.

Thanks.

**Pull request closed without merging**